docs/conditions.md

```diff
@@ -6,50 +6,54 @@ weight: 11
 -->
 # Conditions
 
-This document defines `Conditions` and their capabilities.
+- [Overview](#overview)
+- [Configuring a `Condition`](#configuring-a-condition)
+  - [Specifying the condition `check`](#specifying-the-condition-check)
+  - [Specifying `Parameters`](#specifying-parameters)
+  - [Specifying `Resources`](#specifying-resources)
+- [Code examples](#code-examples)
 
-- [Syntax](#syntax)
-  - [Check](#check)
-  - [Parameters](#parameters)
-  - [Resources](#resources)
-- [Labels and Annotations](#labels-and-annotations)
-- [Examples](#examples)
+## Overview
 
-## Syntax
+A `Condition` resource in Tekton allows you to conditionalize the execution of `Tasks` within a `Pipeline`.
+You define each `Condition` within your [`PipelineRun` definition](pipelineruns.md) and then conditionalize
+each desired [`Task`](tasks.md) in the corresponding `Pipeline` definition. 
 
-To define a configuration file for a `Condition` resource, you can specify the
-following fields:
+The `Condition` resource runs its own container image that executes the logic that evaluates your chosen condition.
+This container runs to completion and must return an exit code value of `0` for the `check` to be successful; all
+other exit codes result in a failure.
```

Comment:
Wdyt about qualifying the meaning of "failure" here?  If I remember right the pipeline doesn't end up failed, but the conditioned task in the pipeline is skipped.

Reply:
the branch of the conditioned task is skipped as well

Reply:
the branch is made up of the dependent tasks -- based on ordering (defined using runAfter) and resource dependencies (e.g. results)

Reply:
the branch of the conditioned task is skipped as well -- the branch is made up of the dependent tasks (based on ordering dependencies defined using runAfter and resource dependencies e.g. results)

Reply:
All great points - reworded to qualify the outcome in the event of an unsuccessful condition check.